io-backend/src/main/java/agh/io/iobackend/model/PlayerState.java

```diff
@@ -7,25 +7,29 @@ public class PlayerState {
 
     private int xCoordinate;
     private int yCoordinate;
-    private final Vector vector;
+    private Vector vector;
 
     public PlayerState(int xCoordinate, int yCoordinate){
         this.xCoordinate = xCoordinate;
         this.yCoordinate = yCoordinate;
         this.vector = new Vector(0,0);
     }
 
-    public void updateVectorAndCoordinates(int xChange, int yChange){
```

Comment:
Dlaczego zrezygnowaliśmy z tej metody - `updateVectorAndCoordinates()`?

Reply:
Znaczy ona jest tylko, że wygląda już inaczej - teraz nie obliczamy na podstawie `xChange` i `yChange` tylko po prostu przepisujemy zmienne podane w `GameController` w tym `MoveRequest`

Reply:
ok, to ma sens. Po prostu przyjmujemy takie dane od Fronta